sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/BulkProcessingOptionsTest.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.batch.PartitionScopeThresholds;
+import org.testng.Assert;
```

Comment:
codestyle: for tests we use assertj assertion.
could you please check other tests as sample